lib/tasks/push-to-space/push-to-space.js

```diff
@@ -199,7 +202,7 @@ export default function pushToSpace ({
       title: 'Importing Content Entries',
       task: wrapTask((ctx, task) => {
         return creation.createEntries(
-          {environment: ctx.environment, skipContentModel},
+          {space: ctx.environment, skipContentModel},
```

Comment:
I did call the space property in export `source` since it can bei either a space or an environment.

I suggest to use `target` here :)

Reply:
I was avoiding editing all the other files, but yeah I guess I have to do it 😅

Reply:
🚀 it! :D